src/Service/Lambda/src/Enum/Runtime.php

```diff
@@ -3,7 +3,7 @@
 namespace AsyncAws\Lambda\Enum;
 
 /**
- * A runtime identifier. For example, `go1.x`.
+ * The runtime environment for the Lambda function.
```

Comment:
why isn't it updated by the bot?

Reply:
Maybe it is. Let's see tomorrow when the bot runs next time.

Reply:
hm.. No update by the bot..

Strange.

Reply:
Ok, I know why.
We don't have documentation for Shapes. We only have documentation when a Shape is referenced by an attribute.

For the Runtime Shape, we have differents documentations:
```
"Runtime": {
      "base": null,
      "refs": {
        "CompatibleRuntimes$member": null,
        "CreateFunctionRequest$Runtime": "<p>The identifier of the function's <a href=\"https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html\">runtime</a>.</p>",
        "FunctionConfiguration$Runtime": "<p>The runtime environment for the Lambda function.</p>",
        "ListLayerVersionsRequest$CompatibleRuntime": "<p>A runtime identifier. For example, <code>go1.x</code>.</p>",
        "ListLayersRequest$CompatibleRuntime": "<p>A runtime identifier. For example, <code>go1.x</code>.</p>",
        "UpdateFunctionConfigurationRequest$Runtime": "<p>The identifier of the function's <a href=\"https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html\">runtime</a>.</p>"
      }
    },
```
That's why: before your PR, the object was only used by `FunctionConfiguration`.
Now it's used by `FunctionConfiguration` and `ListLayerVersionsRequest` and the second definition is used.